deploy/config/keter.yaml

```diff
@@ -1,8 +1,9 @@
 stanzas:
-  - type: webapp
+  - type: background
```

Comment:
@cdepillabout 
To tell the truth, I don't really know the difference between `webapp` and `background`.

If I set `webapp`, keter produced following error after a few minutes since it successfully serve the page. 

```
Error occured when launching bundle kucipong: ensureAlive failed 
CallStack (from HasCallStack): 
  error, called at ./Keter/App.hs:331:14 in keter-1.4.3.2-LCeU5xU7leYDg5Nfn4rLmK:Keter.App
```

I've googled the error message, but no reasonable page found.
By the error message, I thought that `yesod` might provide a API to check if it lives, and keter calls the API at a certain interval.
Though I can not actually confirm it till checking codes of `keter`, the `background` does not check such a kind of checking.